### ISSUE TITLE
On the 'Select tests' dialog, there is not a Scrollbar, and no way to scroll through the List of Tests, the 'SELECT' and 'CANCEL' btns can be unaccessible

### ISSUE THREAD

**Issue by jobs4mvr** [SeleniumHQ/selenium-ide#1013]
## 🐛 Bug Report

When a User has to add new Tests to a Test Suite, a dialog opens with search textbox. When the list of Tests is long, and search does not narrow the list enough for the User to select the desired Test from the List since the List is outside the confines of the Window, AND does not have a scrollbar. 

An offshoot of this issue, and a more serious one, IMHO, is that, even if the User is able to select the desired Tests from the dialog, if the List of Results is long, the 'SELECT' btn (and 'CANCEL' btn) for the dialog is hidden from the screen - pressing the down arrow or the Pg Down key does not scroll the User through the List, leaving the User in a state where he/she cannot submit the selection.

## To Reproduce

(1) Have, say, 40 Test scripts.
(2) Go to the 'Test Suite' module;
(3) Against the name of the 'Test Suite' that you want to add the Tests to, click on the 3 vertically aligned dots;
(4) The 'Select tests' dialog displays;
(5) Do a search, and assume that the List of Tests is still pretty long;
(6) In the next few steps, attempt to  reach Test scripts that are below the screen;
(7) Click on the down arrowhead key on the keyboard and review the outcome - the User is not scrolled down the List;
(8) Click on the 'page-down' key on the keyboarrd and review the outcome - the User is not scrolled down the List;
(9) Try to  reveal a scrollbar on the Right side of the  dialog, and review the  outcome - there is not a scrollbar.
(10) Say, 1 Test is selected from the visible ones in the List - try to submit the dialog by clicking the 'SELECT' btn - the btn is not visible/accessible;
(11) Try to dismiss the dialog by clicking the 'CANCEL' btn - the 'CANCEL' btn is not visible/acccessible; 

## Expected behavior
The User, at all times, should be able to scroll through, select from the 'Select tests' dialog, and use the 'SELECT' btn (or leave/dismiss the dialog using the 'CANCEL' btn) easily, and without issues.

## Project file reproducing this issue (highly encouraged)
UI Issue - not applicable.

Please provide a project file `.side` that reproduces this issue.
UI Issue - not applicable.

## Environment
OS: Windows 10 Pro
Selenium IDE Version:  3.17.0 
Selenium SIDE Runner Version:  Not directly applicable 
Node version: Not Applicable
Browser: Google Chrome
Browser Version: v81.0.4044.92 (Official Build) (64-bit)

Please see attached screenshot with issues highlighted:
![noScroll   noScrollBar_dialogSelectTests](https://user-images.githubusercontent.com/2760106/79166288-cde1cf00-7db2-11ea-811a-bc24210da9aa.PNG)

**Comment by jobs4mvr** [comment]
Closed inadvertently - need to close the other duplicate.

**Comment by jesshhu** [comment]
Hi @jobs4mvr I also experienced this issue, my workaround is just I zoom out the page so I can see the list and the submit button or search for the specific Test so it can have only one item and you can see the submit button. But still, it's good to raised the issue. just passing by, hope it helps.

**Comment by a-tinsmith** [comment]
I also experienced this bug. The environment is similar to the one of the original reporter. Please fix it, the bug is very annoying.

**Comment by jobs4mvr** [comment]
An assumption (possibly) made by the Developers seems to be that the User will always have a mouse with a scroll button - not the case, for example, when I am mobile and do not have a table to setup a separate keyboard and a mouse.

**Comment by jobs4mvr** [comment]
Any update on this issue?

**Comment by jobs4mvr** [comment]
Any update on this issue?

**Comment by pesteban1785** [comment]
Please give us an update on this.  It shouldn't be too hard to add a scrollbar to that modal window so we can get to the bottom of the list where the Add and Cancel buttons are.

**Comment by jobs4mvr** [comment]
It would be nice to get an update and a possible ETA on this...Thank you.

**Comment by zcai2023** [comment]
Is this issue being worked on? I have about 140 tests. I am not able to add all the tests in a test suite, even with the zoom out workaround. And this impacted the Selenium Command-line feature too, without being able to execute all tests.

**Comment by toddtarsi** [comment]
I think this is fixed in v4. You're on your own in v3

**Comment by zcai2023** [comment]
@toddtarsi The latest Selenium IDE for Edge I can get is 3.17.0 (see attached screenshot). How can I upgrade it to v4?
![SeleniumIDE](https://github.com/user-attachments/assets/68ceb130-6c3a-4cd6-bf49-c0917b9bef3a)

**Comment by toddtarsi** [comment]
It's got it's own share of issues, but its semi-actively maintained by me and in beta here:

https://github.com/SeleniumHQ/selenium-ide/releases/tag/v4.0.1-beta.14

**Comment by zcai2023** [comment]
Fair enough :-). Thanks for making this effort! I've just dug out some installation instructions https://github.com/SeleniumHQ/selenium-ide/wiki/Selenium-IDE-v4#how-can-i-migrate-from-v3-to-v4.

**Comment by zcai2023** [comment]
Scroll bar is available in V4. But I can't seem to be able to select multiple tests to drop to the test suite. Maybe I missed something important?

**Comment by toddtarsi** [comment]
Unfortunately, I haven't created batch controls here yet. It's single file drag and drop. Commands support batch copy, cut, and paste using ctrl + shift + c / v, but not the suite UX unfortunately
